content/SCALE/DataProtection/Replication.md

```diff
@@ -0,0 +1,192 @@
+Choosing a remote source requires selecting an SSH connection to that system.
+Expanding the directory browser shows the current datasets or zvols that are available for replication.
+You can select multiple sources or manually type the names into the field.
+
+TrueNAS shows how many snapshots are available for replication.
+We recommend you manually snapshot the sources or create a periodic snapshot task *before* creating the replication task.
+However, when the sources are on the local system and don't have any existing snapshots, TrueNAS can create a basic periodic snapshot task and snapshot the sources immediately before starting the replication. Enabling *Recursive* replicates all snapshots contained within the selected source dataset snapshots.
+
+![TasksReplicationTasksAddSourceSCALE](/images/SCALE/RepAddSourceSCALE.png "Choosing a Local Source")
+
+Local sources can also use a naming schema to identify any custom snapshots to include in the replication.
+Remote sources require entering a *snapshot naming schema* to identify the snapshots to replicate.
+A naming schema is a collection of [strftime](https://www.freebsd.org/cgi/man.cgi?query=strftime) time and date strings and any identifiers that a user might have added to the snapshot name.
+
+{{< /tab >}}
+{{< tab "Destination" >}}
+The destination is where replicated snapshots are stored.
+Choosing a remote destination requires an SSH connection to that system.
+Expanding the directory browser shows the current datasets that are available for replication.
+You can select a destination dataset or manually type a path in the field.
+Zvols cannot be used as a remote replication destination.
+Adding a name to the end of the path creates a new dataset in that location.
+
+![TasksReplicationTasksAddRemoteDestSCALE](/images/SCALE/RepAddDestinationSCALE.png "Replication with Remote Destination")
+
+{{< hint info >}}
+To use encryption when replicating data click the *Encryption* box. Once the box is checked the following additional encryption options will become available:
+
+* *Ecryption Key Format* allows the user to choose between a Hex (base 16 numeral) or Passphrase (alphanumeric) style encryption key. <br>
+* *Store Encryption key in Sending TrueNAS database* allows the user to either store the Encryption key in the sending TrueNAS database (box checked) or choose a temporary location for the encryption key that will decrypt replicated data (box unchecked)
+{{< /hint >}}
+
+{{< /tab >}}
+{{< tab "Security and Task Name" >}}
+{{< hint info >}}
+Using encryption for SSH transfer security is always recommended.
+{{< /hint >}}
+
+In situations where two systems within an absolutely secure network are used for replication, disabling encryption speeds up the transfer.
+However, the data is completely unprotected from malicious sources.
+
+Choosing no encryption for the task is the same as choosing the *SSH+NETCAT* transport method from the advanced options screen.
```

Comment:
No, it's not, SSH+NETCAT is even faster (much faster, but it does not support any encryption)